src/App.tsx

```diff
@@ -1,26 +1,234 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Don't disable linter

Reply:
This is a hint from metor, since otherwise I can't commit